site2/docs/administration-pulsar-manager.md

```diff
@@ -0,0 +1,128 @@
+---
+id: administration-pulsar-manager
+title: The Pulsar Manager
+sidebar_label: Pulsar Manager
+---
+
+Pulsar Manager is a web-based GUI management and monitoring tool that helps administrators and users manage and monitor tenants, namespaces, topics, subscriptions, brokers, clusters, and so on, and supports dynamic configuration of multiple environments.
+
+## Install
+
+The easiest way to use the pulsar manager is to run it inside a [Docker](https://www.docker.com/products/docker) container.
```

Comment:
```suggestion
The easiest way to use the Pulsar Manager is to run it inside a [Docker](https://www.docker.com/products/docker) container.
```

Pulsar Manager is a proper noun, please use the correct capitalization and check all occurrences.